packages/plugin-session-replay-browser/test/jsdom-extended.js

```diff
@@ -0,0 +1,16 @@
+const JSDOMEnvironment = require('jest-environment-jsdom').default; // or import JSDOMEnvironment from 'jest-environment-jsdom' if you are using ESM modules
```

Comment:
Why not use the ESM syntax here instead?

Reply:
Idk this was a copy/paste from a stack overflow post, i think this file doesnt get transpiled